Program.cs

```diff
@@ -10,7 +10,7 @@ private static void Main(string[] args)
         var dirPath = Console.ReadLine();
         while (!Directory.Exists(dirPath))
         {
-            Console.Write("Invalid directory, try again: ");
+            Console.Write("Invalid directory, try again: ")

```

Comment:
Ben, your missing a semi colin
```suggestion
            Console.Write("Invalid directory, try again: ");

```